BrainPortal/spec/controllers/userfiles_controller_spec.rb

```diff
@@ -676,6 +678,26 @@ class << file; attr_reader :tempfile; end
 
 
 
+    describe "export_file_list" do
+      before(:each) do
+        session[:session_id] = 'session_id'
+        allow(controller).to      receive(:current_project).and_return(1010)
```

Comment:
What is this `1010` that the method is retuning? That's not a group....

Reply:
replaced with a group